### DISCUSSION TITLE
Nie działa powiązanie relacji ze skrzynki mailowej

### DISCUSSION THREAD

**Discussion by pablo1pablo** [YetiForceCompany/YetiForceCRM#15319] [How to] [answered]
Witam wszystkich.

Mam pewnien problem z działaniem powiązania bezpośrednio ze skrzynki mailowej .
O tyle ile przycisk lupki działa, i otwiera listę kontaktów, to kliknięcie plusika nie wywołuje okienka z mozliwością wpisania nowego kontaktu do CRM...

Instalacja na apache2, stoi na ubuntu server.

**Comment by mariuszkrzaczkowski** [comment]
![github](https://user-images.githubusercontent.com/10198654/137677384-5cf77884-58eb-478d-85cd-3cba42e2b937.jpg)

**Comment by rskrzypczak** [comment, marked as answer]
If you are using version 6.2 ..
1. Install the latest service pack ([link](https://github.com/YetiForceCompany/YetiForceCRM/releases/tag/6.2.0))
2. Change roundcube version to 0.1.31 ([link](https://github.com/YetiForceCompany/YetiForceCRM/blob/stable/config/version.php#L6)) and upgrade
![image](https://user-images.githubusercontent.com/10221999/137923800-054a175b-c448-4251-b6ea-40756d1e235e.png)
3. Add this fix: https://github.com/YetiForceCompany/YetiForceCRM/commit/f7acbc6801bd40a5868ec10cb9a17fac9e592fc4
4. done

or wait for the release of version 6.3 :)

**Comment by pablo1pablo** [comment]
Dziękuje :-)

Wszystko ładnie działa :-)
